### PR TITLE
Hieu

### PR DESCRIPTION
**Pull request by hieul98**

### CODE REVIEW

File: code/Hieu_L/python/classes_lab.py
Comment (cdmichaelb) [inline code/Hieu_L/python/classes_lab.py]: Good work on the OOP lab!

File: code/Hieu_L/python/classes_lab.py
Comment (cdmichaelb) [inline code/Hieu_L/python/classes_lab.py]: This should either be a return or a print. Not both. Although, it will still work like this. It will return None (the result of the print function) but print because it is still a print function.

**Review by cdmichaelb — CHANGES_REQUESTED** [review]
Pick6 needs work.
It's statistically impossible for the player to net positive every time they play.

Good work on OOP lab.

File: code/Hieu_L/python/classes_lab.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/classes_lab.py]: Let's not ask for the input here. Instead , the deposit function should have two arguments:

def deposit(self, amount):
        self.balance += amount
        return self.balance

File: code/Hieu_L/python/classes_lab.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/classes_lab.py]: Inheritance needs needs correct syntax. You're missing the super method invocation when creating the parallelepiped class. Here's an example:

class Rabbit(Animal):
    def __init__(self, legs, color, speed): ##adding the new attribute speed
        super().__init__(legs, color) ## referencing old attributes legs,color
        self.speed = speed

File: code/Hieu_L/python/classes_lab.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/classes_lab.py]: Missing to call the super method. See comment above.

File: code/Hieu_L/python/weather_api.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/weather_api.py]: You need to be more explicit with the data you want to return. Don't just return the entire JSON object, extract the information you want to display

File: code/Hieu_L/python/weather_api.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/weather_api.py]: You need to be more explicit with the data you want to return. Don't just return the entire JSON object, extract the information you want to display

File: code/Hieu_L/python/weather_api.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/weather_api.py:48]: You need to be more explicit with the data you want to return. Don't just return the entire JSON object, extract the information you want to display

File: code/Hieu_L/python/weather_api.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/weather_api.py]: same as above

File: code/Hieu_L/python/pick6.py
Comment (AlexDeMichieli) [inline code/Hieu_L/python/pick6.py:56]: Missing V2

**Review by AlexDeMichieli — CHANGES_REQUESTED** [review]
Needs changes before approval

**Review by AlexDeMichieli — APPROVED** [review]

**Pull request merged**
